src/access/AccessControl.cpp

```diff
@@ -193,22 +193,107 @@ CHIP_ERROR AccessControl::Finish()
     return retval;
 }
 
-CHIP_ERROR AccessControl::RemoveFabric(FabricIndex fabricIndex)
+CHIP_ERROR AccessControl::CreateEntry(const SubjectDescriptor * subjectDescriptor, FabricIndex fabric, size_t * index,
```

Comment:
Please wait for the PR with the storage code to go in before merging

Reply:
Not an issue any more, the other PR is merged into master and into this PR's branch.